src/Telemetry/Cron_Job.php

```diff
@@ -15,7 +15,7 @@ public function __construct( Telemetry $telemetry, string $plugin_path ) {
 		$this->telemetry = $telemetry;
 
 		// Load ActionScheduler
-		require_once trailingslashit( dirname( $plugin_path ) ) . 'vendor/woocommerce/action-scheduler/action-scheduler.php';
+		require_once trailingslashit( dirname( $plugin_path ) ) . 'lib/vendor/woocommerce/action-scheduler/action-scheduler.php';
```

Comment:
I think we will need to harden up how we are pulling in action-scheduler.  This location has changed a few times in the last few updates.